Makefile

```diff
@@ -179,6 +181,20 @@ runner-build:
 		--cache-from $(CANONICAL_RUNNER_IMAGE):tests-1.24 \
 		-t $(RUNNER_IMAGE):$(RUNNER_IMAGE_TAG_PREFIX)tests-1.24 -f test/e2e/Dockerfile test/e2e
 
+	@echo "building target tests-1.23"
+	@docker build --target tests-1.23 \
+		--cache-from $(RUNNER_IMAGE):$(RUNNER_IMAGE_TAG_PREFIX)builder \
+		--cache-from $(CANONICAL_RUNNER_IMAGE):builder \
+		--cache-from $(RUNNER_IMAGE):$(RUNNER_IMAGE_TAG_PREFIX)tests-1.26 \
+        --cache-from $(CANONICAL_RUNNER_IMAGE):tests-1.26 \
```

Comment:
@timoreimann , nit: indentation issue ? Can you align the `--cache-from` with the other items here for consistency ?

Reply:
Thanks, great catch. Done (and fixed a few 1.26 ones too).